src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -237,7 +238,7 @@
 
 			<!--  Page Navigation Actions  -->
 			<StackPanel
-				Grid.Row="1"
+				Grid.Column="0"
```

Comment:
Was this changed on purpose?

Reply:
Yes, Grid.Row showed a warning (there are no rows defined)

Reply:
Thanks